playwright-e2e/dsm/pages/kitsInfo-pages/kitsSentPage.ts

```diff
@@ -49,6 +52,67 @@ export default class KitsSentPage {
     return await this.kitsTable.getData(columnName);
   }
 
+  /**
+   * Sorts the given column by clicking on it.
+   * Date type columns - first click gives you list starting with dates from long ago; second click starts with recent dates
+   * String type columns - first click gives you A -> Z list; second click gives you Z -> A list
+   * @param columnName name of the column to be sorted
+   */
+  public async sortColumn(opts: {
+    columnName: KitsColumnsEnum,
+    aToZ?: boolean,
+    zToA?: boolean,
+    startWithRecentDates?: boolean,
+    startWithPastDates?: boolean }): Promise<void> {
+    const { columnName, aToZ, zToA, startWithRecentDates, startWithPastDates } = opts;
+
+    const column = columnName as string;
+    const columnSorter = this.page.locator(`//app-shipping//table//th[contains(.,'${column}')]/mfdefaultsorter`);
+    expect(columnSorter, `The column ${column} is not able to be sorted in the Kits Sent page`).toBeTruthy();
+
+    if (aToZ || startWithPastDates) {
+      //Only a single click is needed
+      await columnSorter.click();
+    } else if (zToA || startWithRecentDates) {
+      //Two clicks are needed to get the wanted result
+      await columnSorter.click();
+      await columnSorter.click();
+    }
+  }
+
+  public async getRecentMFBarcodes(): Promise<Locator[]> {
```

Comment:
Is "recent" something that is baked into the UI?  I'm wondering if this method should be something like `getMFBarcodesSince(sinceDay:string)` or something similar.